docs/source/index.html

```diff
@@ -127,7 +127,7 @@ <h2 class="mt-3">Scikit-Learn</h2>
             <h2 class="featurette-heading">Familiar for Python users</h2>
             <h2 class="featurette-subheading"> and easy to get started</h2>
             <p class="lead">Dask uses existing Python APIs and data structures to make it easy to switch between Numpy, Pandas, Scikit-learn to their Dask-powered equivalents. </>
```

Comment:
While we are at it, could you please remove the trailing whitespace on this line?

Reply:
Removed it, and some other spaces as well.